torchgeo/samplers/single.py

```diff
@@ -145,11 +145,8 @@ def __iter__(self) -> Iterator[BoundingBox]:
             if self.dataset.return_as_ts:
                 mint = self.dataset.bounds.mint
```

Comment:
Notebook fails here

Reply:
After merging in main, it no longer complains